admin/local/cli/qserv

```diff
@@ -33,20 +33,23 @@ import sys
 # src/admin/cli/options.py to be imported and used by the qserv command.
 sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "../../../src/admin/python")))
 
-from lsst.qserv.admin.qservCli.qserv_log import log_level_flag, default_log_level
-
+from lsst.qserv.admin.qservCli.qserv_log import log_level_from_args
 
 if __name__ == "__main__":
-    log_level = sys.argv[sys.argv.index(log_level_flag)+1].upper() if log_level_flag in sys.argv else default_log_level
-    logging.basicConfig(
-        level=log_level,
-        format="%(asctime)s %(levelname)-8s %(message)s",
-        datefmt="%Y-%m-%d %H:%M:%S"
-    )
+    success, levelMsg = log_level_from_args(sys.argv)
+    if success:
+        logging.basicConfig(
+            level=levelMsg,
+            format="%(asctime)s %(levelname)-8s %(message)s",
+            datefmt="%Y-%m-%d %H:%M:%S",
+        )
```

Comment:
I agree that initializing logging early is a good idea, but I would still prefer to leave parsing to `click`. Is there a reason why do you need this even before parsing command lie with `click`?

Reply:
I have code that does work while click options are being loaded & initialized, and some of it emits debug log messages. To see those I have to init logging before click is able to parse & init logging. :sadpanda:

Reply:
Maybe comment out those debugging messages? :happypanda: I think for cases like this it should be fine to use some alternative debugging mechanism instead of logging.